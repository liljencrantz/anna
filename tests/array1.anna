
def main(){
    var a = [11,22,33,44,55,66,77];
    print(a.count,"\n");
/*    a[3..6] = [1];
    print(a.count,"\n");
    a=[1];
    a[3..10]=[5];
    a.each(i){print(i)}
    print("\n");
    a=[1,2,3,4,5,6,5,4,3,2,1,2,3,4,5,6,7];
    a[3..12]=[9,9,9];
    a.each(i){print(i)}
    print("\n");
    a=[9,8,7,6,5,4,3,2,1];
    a[3..6:2]=[0,0];
    a.each(i){print(i)}
    print("\n");*/
}
