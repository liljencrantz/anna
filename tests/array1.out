5
1nullnull5
12399954321
987050321
